matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/message/MessageLocationContent.kt

```diff
@@ -49,8 +49,8 @@ data class MessageLocationContent(
         /**
          * Exact time that the data in the event refers to (milliseconds since the UNIX epoch)
          */
-        @Json(name = "org.matrix.msc3488.ts") val unstableTs: Long? = null,
-        @Json(name = "m.ts") val ts: Long? = null,
+        @Json(name = "org.matrix.msc3488.ts") val unstableTimestampAsMilliseconds: Long? = null,
+        @Json(name = "m.ts") val timestampAsMilliseconds: Long? = null,
```

Comment:
Not a blocker, but the un-documented naming rule in the project when the unit is in milliseconds is to use `Millis` suffix. (as per the Android API `System.currentTimeMillis()`)
Applicable to the 2 lines above, and to `getBestTimestampAsMilliseconds()` below

Reply:
Good point! It is easier to read.